js/main.js

```diff
@@ -0,0 +1,504 @@
+let maxBoardSize = 520;													//максимальний розмір дошки у px, висота або ширина
+let tileSize;															//розмір плитки у пікселях
+let tileMoveTime = 90;													//час на пересування плитки
```

Comment:
As I see, this variable doesn't change during whole the code, that's why it must have been const

Reply:
And the same about the rest of the variables